reactos/dll/win32/browseui/shellbrowser.cpp

```diff
@@ -1629,9 +1718,93 @@ void CShellBrowser::UpdateViewMenu(HMENU theMenu)
         SetMenuItemInfo(theMenu, IDM_VIEW_TOOLBARS, FALSE, &menuItemInfo);
     }
     SHCheckMenuItem(theMenu, IDM_VIEW_STATUSBAR, fStatusBarVisible ? TRUE : FALSE);
+}
+
+HRESULT CShellBrowser::BuildExplorerBandMenu()
+{
+    HMENU                                   hBandsMenu;
+    UINT                                    nbFound;
 
-    // TODO: Implement
-    SHEnableMenuItem(theMenu, IDM_VIEW_EXPLORERBAR, FALSE);
+    hBandsMenu = SHGetMenuFromID(fCurrentMenuBar, IDM_VIEW_EXPLORERBAR);
+    if (!hBandsMenu)
+    {
+        OutputDebugString(L"No menu !\n");
+        return E_FAIL;
+    }
+    DSA_DeleteAllItems(menuDsa);
+    BuildExplorerBandCategory(hBandsMenu, CATID_InfoBand, 4, &nbFound);
+    BuildExplorerBandCategory(hBandsMenu, CATID_CommBand, 20, &nbFound);
```

Comment:
Hmm ? It actually appends items to the "Explorer bars" submenu, and adding them at different positions (I saw windows does insert menu items at fixed position thanks to API Monitor). So I guess there is no overwrite (but yes, I can grab separator position and insert other CommBand menu items after)